spras/omicsintegrator1.py

```diff
@@ -118,6 +128,18 @@ def run(edges=None, prizes=None, dummy_mode=None, mu_squared=None, exclude_terms
         bind_path, prize_file = prepare_volume(prizes, work_dir)
         volumes.append(bind_path)
 
+        # 4 dummy mode possibilities:
+        #   1. terminals -> connect the dummy node to all nodes that have been assigned prizes
+        #   2. all ->  connect the dummy node to all nodes in the interactome i.e. full set of nodes in graph
+        #   3. others -> connect the dummy node to all nodes that are not terminal nodes i.e. nodes w/o prizes
+        #   4. file -> custom nodes - connect the dummy node to a specific list of nodes provided in a file
+
+        # add dummy node file to the volume if dummy_mode is not None and it is 'file'
+        if dummy_mode is not None and dummy_mode == 'file':
+                # needs to use dummy node file that was put in the dataset
+                bind_path, dummy_file = prepare_volume(dummy_nodes, work_dir)
```

Comment:
I don't think we have error checking to see if dummy_mode is file and the dummy_nodes file is the default None.